docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
-
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
+What will you get?
+------------------
+.. image:: ../_static/images/integrations/lightning_basic.gif
+   :target: ../_static/images/integrations/lightning_basic.gif
    :alt: PyTorchLightning neptune.ai integration
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
+
+* see experiment as it is running,
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
+
+.. note::
+
+    This integration is tested with ``pytorch-lightning==0.9.0``. See 'Troubleshoot' section for information about other versions.
+
+Quickstart
+----------
+This quickstart will show you how to log PyTorch Lightning experiments to Neptune using ``NeptuneLogger`` (part of the pytorch-lightning library).
+
+Check this public project with example experiments: |project|.
+
+.. tip::
+
+    Follow whatever is most convenient to you:
+
+    1. This page for detailed explanations,
+    2. Open quickstart code in Colab Notebook (badge-link below) and run it as a "`neptuner`" user - zero setup, it just works,
+    3. View quickstart code as Python script on |script|.
+
+|Run on Colab|
+
+Before you start
+^^^^^^^^^^^^^^^^
+**Prerequisites**
+
+* Python 3,
+* Neptune account |register|,
+* ``pytorch-lightning==0.9.0`` and ``torchvision`` installed (lightning |lightning-install|),
+* Minimal familiarity with the PyTorch Lightning.
+
+Installation
```

Comment:
I think I'd change it into Step 0: Install neptune or something. 

...or maybe this should be in the prerequisites. 
Instead of register have this -> with link to installation parts of the docs. 
Remember that people don't have to be registered to use/try it.

Reply:
I moved all to the prerequisited subsection.